connector/components/synchronizer.py

```diff
@@ -55,7 +56,7 @@ def run(self):
     def mapper(self):
         """Return an instance of ``Mapper`` for the synchronization.
 
-        The instanciation is delayed because some synchronisations do
+        The instantiations is delayed because some synchronizations do
```

Comment:
as you're fixing mistakes, you can also choose between `instantiations are` and `instantiation is`